src/apps/queues/models.py

```diff
@@ -0,0 +1,53 @@
+from urllib.parse import urlparse
+
+from django.conf import settings
+from django.db import models
+from pyrabbit.http import HTTPError
+# from django.contrib.sites.models import Site
+
+import uuid
+
+from apps.queues import rabbit
+
+
+class Queue(models.Model):
+    name = models.CharField(max_length=64)
+    vhost = models.UUIDField(unique=True)
+    is_public = models.BooleanField(default=False)
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.DO_NOTHING,
+        related_name='queues',
+        blank=True,
+        null=True
+    )
+    organizers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name='organized_queues',
+        blank=True,
+        help_text="(Organizers allowed to view this queue when they assign their competition to a queue)"
```

Comment:
do we need this? seems superfluous, we don't add help text anywhere else

Reply:
I vote pull it, since we don't use it anywhere else. (I didn't even know it was an option...)